modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -5509,38 +5514,23 @@ type SequenceDigest {
 }
 
 # Sequence-level events.  As commands are issued to execute a sequence, corresponding events are generated.
-type SequenceEvent implements ExecutionEvent {
-  # Identifies the observation to which the event refers
-  location: SequenceEventLocation!
-
-  # Sequence event data
-  payload: SequenceEventPayload!
-
+type SequenceEvent { # implements ExecutionEvent {
```

Comment:
Grackle (?) or me (?) struggle with interfaces.  This would not work when implementing the interface.

Reply:
They aren't actually implemented yet. I will have to remove it from those too unless Rob has any ideas.